OrbitBase/include/OrbitBase/Result.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_RESULT_H_
+#define ORBIT_BASE_RESULT_H_
+
+#include <string_view>
+
+#include "outcome.hpp"
+
+template <typename T, typename E>
+using Result = outcome::result<T, E, outcome::policy::terminate>;
+
+class ErrorMessage final {
+ public:
+  ErrorMessage() = default;
+  explicit ErrorMessage(std::string_view message)
```

Comment:
This looks a bit surprising to me. I feel a std::string as the parameter type would be more appropriate because it's as sink and can be easily moved in place.

Reply:
switched to std::string.